HelpMyStreetFE/HelpMyStreetFE/src/js/account/login.js

```diff
@@ -34,8 +34,11 @@ export const login = async (email, password) => {
             if (authResp.fetchResponse != fetchResponses.SUCCESS) {
                 throw ({ code: "server" });
             }
-
-            if (returnUrl && returnUrl.startsWith("/")) {
+            
+            if (loginURL) {
```

Comment:
ReturnUrl already does something quite similar - but it currently only set when attempting to visit a logged-in page as an unauthorised user.  Could we solve this problem by setting a ReturnUrl in more circumstances (on Community pages, or maybe all pages).